src/config/config_type.rs

```diff
@@ -153,12 +153,28 @@ macro_rules! create_config {
         impl Config {
             $(
             #[allow(unreachable_pub)]
-            pub fn $i(&self) -> $ty {
+            pub fn $i(&self) -> <$ty as StyleEditionDefault>::ConfigType {
                 self.$i.0.set(true);
                 self.$i.2.clone()
             }
             )+
 
+            #[allow(unreachable_pub)]
+            pub fn deafult_with_style_edition(style_edition: StyleEdition) -> Config {
```

Comment:
typo, would need to be updated at call sites too (though I won't try to leave inline comments for those)
```suggestion
            pub fn default_with_style_edition(style_edition: StyleEdition) -> Config {
```

Reply:
Thanks for catching this. Luckily this was a pretty easy fix with find and replace.